scripts-python/create_test

```diff
@@ -121,6 +121,13 @@ formatter_class=argparse.ArgumentDefaultsHelpFormatter
                         help="Number of tasks create_test should perform simultaneously. Default "
                         "will be min(num_cores, num_tests).")
 
+
+    parser.add_argument("--xml_machine",help="Use this machine key in the lookup in testlist.xml")
```

Comment:
You probably want --xml-something instead of --xml_something. You almost never see dashes and underscores combined and is not consistent with any of the other options.